### PR TITLE
IDENTITY-5574: Added form-post response support to Playground2 app

### PR DESCRIPTION
**Pull request by GDRDABARERA**

### CODE REVIEW

File: modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp [hunk 1]
```diff
@@ -76,6 +81,12 @@
             .setResponseType(authzGrantType)
             .setScope(scope);
 
+    if (formPostMode){
```
Comment (madurangasiriwardena) [inline modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp]: Formatting issue.

File: modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp [hunk 2]
```diff
@@ -33,6 +34,10 @@
         if(usePKCE) {
             session.setAttribute(OAuth2Constants.OAUTH2_USE_PKCE, usePKCE);
         }
+        boolean formPostMode = formPostParameter != null && "yes".equals(formPostParameter);
+        if (formPostMode){
```
Comment (madurangasiriwardena) [inline modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp]: Formatting issue.

File: modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp [hunk 3]
```diff
@@ -33,6 +34,10 @@
         if(usePKCE) {
             session.setAttribute(OAuth2Constants.OAUTH2_USE_PKCE, usePKCE);
         }
+        boolean formPostMode = formPostParameter != null && "yes".equals(formPostParameter);
```
Comment (DMHP) [inline modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp]: Shall we define a private constant for "yes" here

File: modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp [hunk 4]
```diff
@@ -76,6 +81,12 @@
             .setResponseType(authzGrantType)
             .setScope(scope);
 
+    if (formPostMode) {
+        oAuthPKCEAuthenticationRequestBuilder.setParameter(OAuth2Constants.OAUTH2_RESPONSE_MODE, OAuth2Constants.OAUTH2_FORM_POST);
+    }
+
```
Comment (ashensw) [inline modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp]: It's better to remove unnecessary new lines.

Comment (GDRDABARERA) [reply modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp]: Fix with https://github.com/wso2/product-is/pull/1369

File: modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp (hunk 3 above)
Comment (GDRDABARERA) [reply modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp]: Fix with https://github.com/wso2/product-is/pull/1369

File: modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp [hunk 5]
```diff
@@ -76,6 +81,12 @@
...
+
+//build the new response mode with form post
```
Comment (omindu) [inline modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp]: Fix formatiing

Comment (GDRDABARERA) [reply modules/samples/oauth2/playground2/src/main/webapp/oauth2-authorize-user.jsp]: Fixed with https://github.com/wso2/product-is/pull/1375

**Pull request merged**